### DISCUSSION TITLE
Rationale behind clearing Quarantined items?

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#481] [Q&A] [answered]
> Clears Quarantined items after 1 day instead of the default behavior of keeping them indefinitely.

What is the rationale behind this? If I'm understanding correctly, it would be safer to keep it as to prevent re-infection.

**Comment by HotCakeX** [comment, marked as answer]
Microsoft Defender knows what you have is malware via file hash and other characteristics, they are [communicated back to the cloud too if necessary](https://learn.microsoft.com/en-us/defender-endpoint/configure-block-at-first-sight-microsoft-defender-antivirus#how-it-works), so it doesn't need the actual file to exist. Quarantine means keeping something dangerous/potentially dangerous in a non-executable part of your system. Removing quarantined files after 1 day instead of keeping them forever because they might still pose a danger and somehow find a way to exploit an unknown vulnerability that we don't know about yet, so not taking any chances.

It definitely doesn't prevent reinfection. If that was the case, then we'd have to keep a copy of every malware in the world in our computers so we wouldn't get infected/re-infected.

At the end of the day, it's defense in depth strategy.

More info: https://learn.microsoft.com/en-us/defender-endpoint/cloud-protection-microsoft-antivirus-sample-submission
